examples/ffi-struct/struct.pony

```diff
@@ -0,0 +1,37 @@
+"""
+Example of the syntax used to pass structs in FFI calls.
+First, build the C file as a library, then build the Pony program.
+"""
+
+use "lib:ffi-struct"
+
+use @modify_via_outer[None](s: Outer)
+use @modify_inner[None](s: Inner)
+
+// Unlike classes, Pony structs have the same binary layout as C structs and
+// can be transparently used in C functions
```

Comment:
It might be useful to mention the downsides of structs (that their lack of a vtable means they cannot be in union types or implement traits/interfaces).

However, I'll leave it up to you.